src/controllers/rest/impl/SubmissionController.ts

```diff
@@ -54,7 +54,7 @@ export class SubmissionController extends BaseRestController {
                                 @PathParams("id") submissionId: number,
                                 @QueryParams("link") youtubeLink?: string): Promise<unknown> {
         await this.submissionService.addYoutubeToSubmission(submissionId, youtubeLink ?? null);
-        return super.doSuccess(res, `Submission youtube link has been assigned`);
+        return super.doSuccess(res, `Submission Youtube link has been assigned.`);
```

Comment:
YouTube

I think the product name has a capital Y. at least my IDE complains about it